<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RvPlatformHistoryBehaviorPrefWidget</class>
 <widget class="QWidget" name="WkfPlatformHistoryPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Platform History - Behavior Trace Line</string>
  </property>
  <layout class="QVBoxLayout" name="layout_1">
   <item>
    <widget class="QGroupBox" name="mBehaviorColors">
     <property name="title">
      <string>Behavior Colors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This color is used when there is no active behavior, including for platforms without behavior trees.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>No Behavior Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="UtQtColorButton" name="mNoBehaviorColorButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This color is used when there is no active behavior, including for platforms without behavior trees.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="mBehaviorColorTable">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The colors of the trace lines given which behavior node is active.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <column>
         <property name="text">
          <string>Behavior Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Behavior Color</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout1">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a behavior from an existing behavior tree that is not currently colorized.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Behavior Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mBehaviorName_tx">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a behavior from an existing behavior tree that is not currently colorized.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mAddBehavior">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a behavior from an existing behavior tree that is not currently colorized.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mRemoveBehavior">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove the selected behavior.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtColorButton</class>
   <extends>QPushButton</extends>
   <header>UtQtColorButton.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops/>
 <resources/>
 <connections/>
</ui>
